tests/test_plugin.py

```diff
@@ -88,6 +110,89 @@ def test_not_required_prebuild_plugin_failure(docker_tasker, runner_type, requir
         results = runner.run()
     assert workflow.plugin_failed is required
 
+@pytest.mark.parametrize('success1,success2', [
+    (True, True),
+    (True, False),
+    (False, True),
+    (False, False),
+])
+@pytest.mark.parametrize('stop', [
+    True,
+    False,
+])
```

Comment:
(It actually tests that the plugins runner does not raise an exception, so that's fine.)